src/core/core.helpers.js

```diff
@@ -486,7 +486,7 @@ module.exports = function() {
 	 */
 	helpers._getParentNode = function(domNode) {
 		var parent = domNode.parentNode;
-		if (parent && parent.host) {
+		if (parent && Object.prototype.toString.call(parent) === '[object ShadowRoot]') {
```

Comment:
I think I prefer `if (parent && parent.toString() === '[object ShadowRoot]') {` because it's better minified.

Reply:
@simonbrunel done